ReactWindows/ReactNative.Net46/Views/View/ReactViewManager.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Data;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using ReactNative.Views.View;
+
+namespace ReactNative.Net46.Views.View
+{
+    /// <summary>
+    /// WPF-specifics of the RCTView implementation.
+    /// </summary>
+    public class ReactViewManager: BaseViewManager
+    {
+        /// <summary>
+        /// Enables the Canvas as a drop target.
+        /// </summary>
+        [ReactProp("allowDrop")]
+        public void SetAllowDrop(BorderedCanvas view, bool allowDrop)
+        {
+            throw new NotImplementedException("The allowDrop property is not available on this platform");
+        }
```

Comment:
We currently don't have any way of differentiating JavaScript for WPF vs. UWP (other than maybe a version check). I think the best approach for now is to create props that silently ignore settings. There was some discussion at one point about trying to expose a yellow box API to native so we could warn.

Reply:
Fixed